Gemfile.lock

```diff
@@ -499,8 +500,7 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
-  arm64-darwin-22
-  arm64-darwin-23
+  x86_64-darwin-22
```

Comment:
Please don't delete `arm64-darwin-23` -- that's my M3.

Reply:
How odd! We could get into a weird back and forth where every time I run bundle, it adds it back in, every time you run bundle it removes it :)